leftwm-core/src/layouts/mod.rs

```diff
@@ -54,6 +54,12 @@ impl Default for Layout {
     }
 }
 
+impl Default for &Layout {
+    fn default() -> Self {
+        &Layout::MainAndVertStack
+    }
+}
+
```

Comment:
This is kinda redundant. With returning the current layout this won't be needed anyway.